walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,67 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+  font-family: Arial, sans-serif;
+  padding: 4px;
+  text-align: center;
 }
 
-.image-thumbnails {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+h1 {
+  text-align: center;
+  font-weight: normal;
+}
+
+nav {
```

Comment:
Consider giving class names or ids to these elements rather than using the HTML tag name in your code, with an exception for the "base styles" of your page (i.e. use a particular font by default, remove underlines for all links, etc.). 

Class names or ids that are short and descriptive act as self-documentation for code, and are more performant to use as accessors than relying on child accessors. For the selectors you have below consider something like:
- `nav` --> `#nav`
- `nav ul` --> `.navLinks`
- `nav ul li` --> `.navLinkContainer`
- `nav ul li a` --> `.navLink`
- etc.